main.go

```diff
@@ -261,7 +261,22 @@ func mainErr(args []string) error {
 			}
 
 			flagSeed = "random;" + base64.StdEncoding.EncodeToString(seed)
+		} else {
+			seed, err := base64.RawStdEncoding.DecodeString(flagSeed)
+			if err != nil {
```

Comment:
is this needed? can we not decode a base64 string with or without padding in a single pass?

if it's not possible, then I argue that we just do it without padding, for the reason outlined in the issue - the `=` character is kind of problematic for flags, and there really isn't a need for padding here.

Reply:
Yes it is needed, I could however just trim all "=" characters from the end.

Reply:
:) I like that hack, at least more than not supporting any padding at all.